src/dag-node.js

```diff
@@ -82,6 +82,9 @@ module.exports = class DAGNode {
 
   // addNodeLink - adds a DAGLink to this node that points to node by a name
   addNodeLink (name, node) {
+    if (!name) {
```

Comment:
That is what I understood when we first made the module. But it seems that go-ipfs supports both empty string and empty buffer, and that is why our sort failed on ipfs-geo-ip, since the dag nodes that were returned didn't have any name property if their name properties were empty.